network-config-analyzer/TcpLikeProperties.py

```diff
@@ -277,19 +192,11 @@ def contained_in(self, other):
         :return: Whether all (source port, target port) pairs in self also appear in other
         :rtype: bool
         """
-        if not self.cubes_set.contained_in(other.cubes_set):
-            return False
-        for port_name in self.named_ports:
-            if port_name not in other.named_ports:
-                return False
-            if not self.named_ports[port_name].contained_in(other.named_ports[port_name]):
-                return False
-        for port_name in other.excluded_named_ports:
-            if port_name not in self.excluded_named_ports:
-                return False
-            if not other.excluded_named_ports[port_name].contained_in(self.excluded_named_ports[port_name]):
-                return False
-        return True
+        assert not self.named_ports
```

Comment:
can replace the 4 assertions with the following:
```
assert not self.has_named_ports() and not other.has_named_ports()
```

Reply:
Done